src/server.ts

```diff
@@ -47,100 +47,141 @@ const startServer = async () => {
+      // TO-DO: Authentication
+      const data = await request.files()
+      const fileData = await saveFiles(data, objectKeysToSnakeCase(request.query))
+      reply.send({ success: true, fileData })
+    })
+
+    // Unique name/email/organisation check
+    server.get('/check-unique', async (request: any, reply) => {
+      const { type, value, table, field } = request.query
+      if (value === '' || value === undefined) {
+        reply.send({
+          unique: false,
+          message: 'Value not provided',
+        })
+        return
+      }
+      let tableName, fieldName
+      switch (type) {
+        case 'username':
+          tableName = 'user'
+          fieldName = 'username'
+          break
+        case 'email':
+          tableName = 'user'
+          fieldName = 'email'
+          break
+        case 'organisation':
+          tableName = 'organisation'
+          fieldName = 'name'
+          break
+        default:
+          if (!table || !field) {
+            reply.send({
+              unique: false,
+              message: 'Type, table, or field missing or invalid',
+            })
+            return
+          } else {
+            tableName = table
+            fieldName = field
+          }
+      }
+      try {
+        const isUnique = await DBConnect.isUnique(tableName, fieldName, value)
+        reply.send({
+          unique: isUnique,
+          message: '',
+        })
+      } catch (err) {
+        reply.send({ unique: false, message: err.message })
+      }
+    })
+
+    done()
+  }
 
   server.get('/', async (request, reply) => {
     console.log('Request made')
     return 'This is the response\n'
   })
 
-  // Unique name/email/organisation check
-  server.get('/check-unique', async (request: any, reply) => {
-    const { type, value, table, field } = request.query
-    if (value === '' || value === undefined) {
-      reply.send({
-        unique: false,
-        message: 'Value not provided',
-      })
-      return
-    }
-    let tableName, fieldName
-    switch (type) {
-      case 'username':
-        tableName = 'user'
-        fieldName = 'username'
-        break
-      case 'email':
-        tableName = 'user'
-        fieldName = 'email'
-        break
-      case 'organisation':
-        tableName = 'organisation'
-        fieldName = 'name'
-        break
-      default:
-        if (!table || !field) {
-          reply.send({
-            unique: false,
-            message: 'Type, table, or field missing or invalid',
-          })
-          return
-        } else {
-          tableName = table
-          fieldName = field
-        }
-    }
-    try {
-      const isUnique = await DBConnect.isUnique(tableName, fieldName, value)
-      reply.send({
-        unique: isUnique,
-        message: '',
-      })
-    } catch (err) {
-      reply.send({ unique: false, message: err.message })
-    }
-  })
+  server.register(api, { prefix: '/api' })
```

Comment:
Just wondering -- what's the advantage of all routes starting with `/api`? Convention?